capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="event.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="hidden" id="organizationID" name="organizationID"> 
+        <br><br>
+
+        <label for="organizationName"> Organization Name: </label>
+        <input type="text" id="organizationName" name="organizationName">
+        <br><br>
+
+        <label for="university"> University: </label>
+        <input type="hidden" id="university" name="university">
+        <br><br>
+
+        <label for="eventTitle"> Event Title: </label>
+        <input type="text" id="eventTitle" name="eventTitle">
+        <br><br>
+
+        <label for="eventDateTime"> Event Date and Time: </label>
+        <input type="datetime-local" id="eventDateTime" name="eventDateTime">
```

Comment:
I did not know about `datetime-local`! It's neat. It seems to not be yet supported in Firefox or Opera, but it's great for prototyping.
https://caniuse.com/#search=datetime-local